AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/ForumMessageRoutes.java

```diff
@@ -32,7 +33,9 @@ public ForumMessage createForumMessage(Request req, Response res) {
         ForumMessage forumMessageModel = JsonRequestUtils.readBodyAsType(req, ForumMessage.class);
         permissionsVerifier.verifyisAgentorAdmin(req);
 
-        forumMessageModel.setUserId(req.attribute("user_id"));
+        User user = new User();
```

Comment:
This is just a fudge to get the build to work - we should actually set the user with the req.attribute obviously!